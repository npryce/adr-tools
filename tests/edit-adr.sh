adr init architecture-log
adr new First Record

export VISUAL
export EDITOR

if adr edit
then
  echo ERROR: should have failed
fi

VISUAL=
EDITOR=fake-editor
adr edit

VISUAL=fake-visual
EDITOR=
adr edit

VISUAL=fake-visual
EDITOR=fake-editor
adr edit

adr edit 1
