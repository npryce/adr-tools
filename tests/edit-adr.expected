adr init architecture-log
architecture-log/0001-record-architecture-decisions.md
adr new First Record
architecture-log/0002-first-record.md

export VISUAL
export EDITOR

if adr edit
then
  echo ERROR: should have failed
fi
ERROR: editor not configured

VISUAL=
EDITOR=fake-editor
adr edit
EDITOR architecture-log/0002-first-record.md

VISUAL=fake-visual
EDITOR=
adr edit
VISUAL architecture-log/0002-first-record.md

VISUAL=fake-visual
EDITOR=fake-editor
adr edit
VISUAL architecture-log/0002-first-record.md

adr edit 1
VISUAL architecture-log/0001-record-architecture-decisions.md
